### PR TITLE
[ADD] product taxes group that replace account_product_fiscal_classification

### PR DESCRIPTION
**Pull request by legalsylvain**
The module 'product_taxes_group' manages taxes classification like 'account_product_fiscal_classification' but with many2one field (instead of property).
You can read a whole comparison between the two modules here : Ref : https://github.com/OCA/account-financial-tools/pull/32

@rvalyi : I'm waiting for your review.

### CODE REVIEW

**Comment by legalsylvain** [comment]
Ping. CC : @rvalyi, @sebastienbeau.

**Comment by danimaribeiro** [comment]
Hey, in the other discussion you told:
"close this V7 PR. In V7, 'account_product_fiscal_classification' is the official one ;
realize a new PR in V8, (as said 'account_product_fiscal_classification' is unported) keeping my code, but renaming module, fields and classes by the name used in account_product_fiscal_position' as much as possible."

Is it possible for you to change the names?
There are a lot of code here in Brazil that depends on the old module:
https://github.com/odoo-brazil/l10n-brazil/blob/8.0/l10n_br_account_product/account_product_fiscal_classification_view.xml
https://github.com/odoo-brazil/l10n-brazil/blob/8.0/l10n_br_account_product/account_product_fiscal_classification.py

We ported to v8 here: https://github.com/odoo-brazil/account-fiscal-rule

**Comment by legalsylvain** [comment]
Hi, since 3 months I didn't receive any review. If you're interested to review it, yes I'll do the according changes I mentionned.

Regards.

File: product_taxes_group/model/tax_group.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+from openerp.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class TaxGroup(models.Model):
+    """Group of customer and supplier taxes.
+    This group is linked to a product to select a group of taxes in one
+    time."""
+    _name = 'tax.group'
```
Comment (danimaribeiro) [inline product_taxes_group/model/tax_group.py:35]: 'account.product.fiscal.classification'

File: product_taxes_group/model/tax_group.py [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    def _get_product_tmpl_ids(self):
+        for rec in self:
+            rec.product_tmpl_ids = self.env['product.template'].search([
+                ('tax_group_id', '=', rec.id), '|', ('active', '=', False),
+                ('active', '=', True)])
+
+    # Field Section
+    name = fields.Char(
+        size=_MAX_LENGTH_NAME, required=True, select=True)
+
```
Comment (danimaribeiro) [inline product_taxes_group/model/tax_group.py:58]: Could you add "description = fields.char('Description', size=64)" ?

File: product_taxes_group/model/tax_group.py [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+    product_tmpl_qty = fields.Integer(
+        string='Products Quantity', compute=_get_product_tmpl_qty)
+
+    supplier_tax_ids = fields.Many2many(
+        'account.tax', 'product_supplier_tax_rel',
+        'group_id', 'tax_id', string='Supplier Taxes', domain="""[
+            ('company_id', '=', company_id),
+            ('parent_id', '=', False),
+            ('type_tax_use', 'in', ['purchase', 'all'])]""")
+
```
Comment (danimaribeiro) [inline product_taxes_group/model/tax_group.py:83]: The old ones are: 'sale_base_tax_ids' and 'purchase_base_tax_ids'.
And also think it needs to change the relation table names.

**Comment by legalsylvain** [comment]
Oups, I did it again. (bad rebase).

**Comment by danimaribeiro** [comment]
@legalsylvain I sent to your branch the portuguese translations to this module.
If you don't have time, I can change the names and test it here.

**Comment by legalsylvain** [comment]
Hi @danimaribeiro. No thanks, I'm working on that. 
I'll realize the following changes : 
-> module name changes ; 
-> class name changes ; 
-> field name changes, **except** 
product_template.property_fiscal_classification --> product_template.fiscal_classification because, it is not a property.
If you want to glue this new module, with your module, you can create a glue module with a field property_fiscal_classification related to field fiscal_classification.

Keep in touch. 

Kind regards and thanks for your review.

**Comment by danimaribeiro** [comment]
It sounds good.

**Comment by legalsylvain** [comment]
Duplicate with #28. I close this PR.
Kind regards.

**Pull request closed without merging**